environment/charge-oss-mxfp4.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
```

Comment:
@chrundle Let's put this file in our local project space for now and we can then add it to a gitlab internal project to keep it version controlled.